include/efsw/efsw.hpp

```diff
@@ -90,27 +92,54 @@ typedef Actions::Action Action;
 namespace Errors {
 
 enum Error {
+	NoError = 0,
 	FileNotFound = -1,
 	FileRepeated = -2,
 	FileOutOfScope = -3,
 	FileNotReadable = -4,
-	FileRemote = -5, /** Directory in remote file system ( create a generic FileWatcher instance to
-						watch this directory ). */
-	Unspecified = -6
+	/// Directory in remote file system
+	/// ( create a generic FileWatcher instance to watch this directory ).

```

Comment:
You forgot the first part of the comment: `Directory in remote file system`

Reply:
Did I? I changed the /** style comment for FileRemote to a /// comment, as used in (most of) the other code and added a comment for the new value WatherFailed.